### PR TITLE
Fixes: Media Details Scrolling issue and use Constraint layout instead of traditional layouts in media_details_fragment. 

### PR DESCRIPTION
**Pull request by gconnect**
I am done with the work.

### CODE REVIEW

File: app/src/main/res/layout/fragment_media_details.xml [hunk 1]
```diff
@@ -8,304 +8,286 @@
...
-        android:fillViewport="true"
-        android:fitsSystemWindows="true"
-        >
-
-        <LinearLayout
+        android:layout_height="match_parent"
+        app:layout_constraintTop_toBottomOf="@id/layout_toolbar"
+        android:orientation="vertical">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_media_details.xml]: @gconnect can we avoid nesting in the layouts. Please take a look at this article https://medium.com/@vleonovs8/how-to-avoid-nested-containers-inside-constraintlayout-3a4d020edd0f

File: app/src/main/res/layout/fragment_media_details.xml [hunk 2]
```diff
@@ -8,304 +8,286 @@
...
+            android:id="@+id/categories_divider"
+            app:layout_constraintTop_toBottomOf="@id/rv_category_list"
+            android:layout_marginLeft="@dimen/margin_8dp"
+            android:layout_marginTop="@dimen/margin_16dp"            android:layout_width="match_parent"
+            android:layout_height="2dp"
+            android:background="@color/colorGrayLine" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_media_details.xml]: Avoid nesting.

Comment (gconnect) [reply app/src/main/res/layout/fragment_media_details.xml]: I will fix it now.

**Comment by ujjwalagrawal17** [comment]
@gconnect let me know when the nested constraint layouts are removed

**Comment by gconnect** [comment]
I have fixed it

**Comment by ujjwalagrawal17** [comment]
<table>
<tr>
<td>

WhatsApp Image 2019-10-03 at 6 13 59 PM
**Before**
</td>
<td>

WhatsApp Image 2019-10-03 at 6 13 58 PM
**Before**

</td>
</tr>

<tr>
<td>

image
**After**

</td>
<td>

image
**After**

</td>
</tr>
</table>

**Comment by ujjwalagrawal17** [comment]
@gconnect there are some issues in the PR. 
1. The name of the file seems to be shifted to the last.
2. The padding between layouts is relatively small compared to the previous version. We need to increase it. 
3. the text size seems to be increased.

**Comment by ujjwalagrawal17** [comment]
Also, the indentation of Author, license, date layouts is not same.

**Comment by ujjwalagrawal17** [comment]
Also, the old design looked good to me. We don't need expandable layouts as of now.

**Comment by ujjwalagrawal17** [comment]
I have updated the title of the PR.

**Comment by gconnect** [comment]
okay fine. Thanks

**Comment by gconnect** [comment]
Okay, I will remove the expandables and revert the sizes back to previous
and also work on the corrections. Thanks for checking.

**Comment by gconnect** [comment]
Modifications have been made. Please review. Thanks

File: app/src/main/res/drawable/ic_expand_more_black_24dp.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ujjwalagrawal17) [inline app/src/main/res/drawable/ic_expand_more_black_24dp.xml]: Can you remove this file as it is unused?

File: app/src/main/res/drawable/ic_expand_less_black_24dp.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ujjwalagrawal17) [inline app/src/main/res/drawable/ic_expand_less_black_24dp.xml]: Can you remove this file as it is unused?

File: app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -18,4 +21,11 @@
     <dimen name="text_size_14sp">14sp</dimen>
     <dimen name="padding_2dp">2dp</dimen>
     <dimen name="margin_140">140dp</dimen>
+    <dimen name="margin_48dp">48dp</dimen>
```
Comment (ujjwalagrawal17) [inline app/src/main/res/values/dimens.xml]: Can you remove unused dimensions that you have added?

**Comment by gconnect** [comment]
Okay will do that now

**Comment by gconnect** [comment]
Just check the work again realized that the dimens I in use by some other
activity layout. I wasn't the one that added them. Except for the 48dp
which am using.

**Comment by gconnect** [comment]
Please review. So I can move on to the next task.

**Comment by ujjwalagrawal17** [comment]
Except dimensions, there are drawables that you added that are not removed yet.

**Comment by ujjwalagrawal17** [comment]
ic_expand_more_black_24dp.xml , ic_expand_less_black_24dp.xml

**Comment by gconnect** [comment]
Will push the updated work soon.

**Comment by gconnect** [comment]
I have removed unused drawable.

**Comment by gconnect** [comment]
Please consider closing this too, If there are no other changes you want me to make

**Comment by gconnect** [comment]
Please consider closing this

**Comment by ujjwalagrawal17** [comment]
@gconnect there should be no changes in activity_welcome in this PR. Can you revert the changes?

**Comment by gconnect** [comment]
You know I have started working on the onboarding. I can revert my changes to splash screen but it will affect the flow of the on-boarding

**Comment by ujjwalagrawal17** [comment]
Also tab_selector drawable is Missing so we can't merge this PR now. As people won't be able to run master branch then.

**Comment by ujjwalagrawal17** [comment]
You can work on this PR after onboarding or before that. No problem on that.

**Comment by gconnect** [comment]
I was the one that added the tab selector previously. And am no longer
using it. That's while I removed it as one of the unused drawables

**Comment by ujjwalagrawal17** [comment]
This is not needed as issue is already resolved by your other PR !

**Pull request closed without merging**